src/controllers/mainController.ts

```diff
@@ -86,6 +86,7 @@ export default class MainController implements vscode.Disposable {
         includeEstimatedExecutionPlanXml: false,
         includeActualExecutionPlanXml: false,
     };
+    private _actualPlanStatuses: string[] = [];
```

Comment:
Do you think it would make more sense to track these in queryResultsController, and call into that from here?